src/benchmark/bm.cr

```diff
@@ -44,13 +44,13 @@ module Benchmark
       end
 
       # Reports a single benchmark unit.
-      def report(label = " ", &block)
+      def report(label : String = " ", &block) : Nil
```

Comment:
This was actually leaking `@reports` below, I manually set this to `Nil` as I don't think that was intended.